src/helpers.js

```diff
@@ -117,20 +117,24 @@ function dotPathToHash(entry, target = {}, options = {}) {
 /**
  * Takes a `source` hash and makes sure its value
  * is pasted in the `target` hash, if the target
- * hash has the corresponding key (or if `keepRemoved` is true).
+ * hash has the corresponding key (or if `options.keepRemoved` is true).
  * @returns An `{ old, new, mergeCount, pullCount, oldCount }` object.
  * `old` is a hash of values that have not been merged into `target`.
  * `new` is `target`. `mergeCount` is the number of keys merged into
  * `new`, `pullCount` is the number of context and plural keys added to
  * `new` and `oldCount` is the number of keys that were either added to `old` or
- * `new` (if `keepRemoved` is true and `target` didn't have the corresponding
+ * `new` (if `options.keepRemoved` is true and `target` didn't have the corresponding
  * key).
  */
-function mergeHashes(source, target, keepRemoved = false) {
+function mergeHashes(source, target, options = {}) {
   let old = {}
   let mergeCount = 0
   let pullCount = 0
   let oldCount = 0
+
+  const keepRemoved = options.keepRemoved || false
```

Comment:
During testing I noticed the `keepRemoved` option is no longer respected. We have `keepRemoved: true` in a config file and these entries are removed, anyway.

Caused by following lines not correctly calling mergeHashes with options object:
* helpers.js:143
* transform.js:197

Reply:
Good catch thanks!

Reply:
Done and added two tests for it